lib/pages/home/home.dart

```diff
@@ -1,90 +1,80 @@
 import 'package:animated_text_kit/animated_text_kit.dart';
 import 'package:flutter/material.dart';
 import 'package:soundspace/constants/style.dart';
+import 'package:soundspace/pages/account/account.dart';
 import 'package:soundspace/widgets/custom_text.dart';
 
-/*
-class ImageHandler extends StatefulWidget {
-  const ImageHandler({Key? key}) : super(key: key);
-
-  @override
-  _ImageHandlerState createState() => _ImageHandlerState();
-}
-
-// This widget is tracking its own state
-class _ImageHandlerState extends State<ImageHandler> {
-  // private vars
-  final _formKey = GlobalKey<FormState>();
-
-  @override
-  Widget build(BuildContext context) {
-    return Form(
-      child: Column(
-        children: <Widget>[
-          // TextFeild
-          TextFormField(
-            decoration: InputDecoration(
-              hintText: 'Insert Text Here',
-              border: OutlineInputBorder(),
-              contentPadding: EdgeInsets.all(20.0),
-            ),
-            validator: (value) {
-              if (value == null || value.isEmpty) {
-                return 'Please Enter Text';
-              }
-              // This is the actual return that will use 'value'
-              return null;
-            },
-          ),
-          Padding(
-            padding: const EdgeInsets.symmetric(vertical: 16.0),
-            child: ElevatedButton(
-              onPressed: () {
-                // Validate will return true if form is valid
-                if (_formKey.currentState!.validate()) {
-                  // form is invalid
-                  //Proccess data
-                }
-              },
-              child: const Text('Submit'),
-            ),
-          ),
-        ],
-      ), // Column
-    );
-  }
-}
-*/
-
 class HomePage extends StatelessWidget {
```

Comment:
This is probably going to evolve into a statefull widget. This comment is here more for record than anything.